packages/cli/src/commands/org/config/orgConfigBase.ts

```diff
@@ -0,0 +1,137 @@
+import {Command, flags} from '@oclif/command';
+import {cli} from 'cli-ux';
+import {ReadStream} from 'fs';
+import {dedent} from 'ts-dedent';
+import {cwd} from 'process';
+import {Config} from '../../../lib/config/config';
+import {Project} from '../../../lib/project/project';
+import {SnapshotFactory} from '../../../lib/snapshot/snapshotFactory';
+import {platformUrl} from '../../../lib/platform/environment';
+import {Snapshot} from '../../../lib/snapshot/snapshot';
+import {red, green} from 'chalk';
+import {normalize} from 'path';
+
+export interface CustomFile extends ReadStream {
+  type?: string;
+}
+
+export default abstract class OrgConfigBase extends Command {
```

Comment:
I created a custom base class so the same code can be reused in the preview and push commands.
Even if I am not a big fan of inheritance, this approach seems to be the most appropriate for this case.

Before that, I tried to simply call the preview methods from the Push command. It worked, but the Oclif team advises against calling [commands programatically](https://oclif.io/docs/running_programmatically).

I also tried moving the common code in a Facade kind of class, but it requires passing a bunch of arguments and flags specific to commands.

Anyhow, let me know if you think that is not the right approach.